### ISSUE TITLE
Update to 0.13

### ISSUE THREAD

**Issue by byteandahalf** [MrCrayfish/MrCrayfishFurnitureMod-MCPE#43]
When I get home from work tonight I'll start updating this to 0.13 build 4.

**Comment by MrCrayfish** [comment]
That sounds like an excellent plan. I will tank adding content once you do :)

**Comment by skullkeeper94** [comment]
:0 no... way...

**Comment by byteandahalf** [comment]
Yes way. So far I've got the Block header updated and I've renamed all TileTessellators to BlockTessellator. I'm also going to add the new Item header and fix AABB (since in 0.13 they moved book isEmpty to the beginning of the strict rather than the end)

**Comment by RandomAltThingy** [comment]
@byteandahalf Make a PR so people (like me) can contribute to your fork and your PR.

**Comment by byteandahalf** [comment]
I have wrote access to this mod, so I don't need to make a PR, I can commit directly.

**Comment by RandomAltThingy** [comment]
Oh yeah I forgot.
